web-dev/src/components/BoxArtList/index.js

```diff
@@ -0,0 +1,20 @@
+import React from "react"
+import "./box-art-list.css"
+import BoxArt from "./BoxArt.js";
+
+
+
+const BoxArtList = () => {
+    return (
+        <>
+            <ul className="list-group wd-trending-bg-color py-3">
+                <li className="list-group-item fw-bold fs-4">Trending Now</li>
```

Comment:
I will refactor this "Trending Now" and move it outside of the BoxArt and into the HomePageComponent. I think it's nicer since it gives a little title/description to the <BoxArt /> call.